composables/useIntl.ts

```diff
@@ -103,7 +103,8 @@ const formatCurrency = (value: string, symbol: string) => {
     return ''
   }
 
-  const currencyValue = Number.parseFloat(fromWei(value)) * currencyMultiplier
+  const currencyValue =
+    Number.parseFloat(fromWei(value, 'ether')) * currencyMultiplier // TODO use BN to do calculations
```

Comment:
Is this TODO comment a reminder for the future? or did you forget here something?

Reply:
For future (as more proper way), right now it doesn't impact anything.